design/baremetal-operator/reboot-interface.md

```diff
@@ -138,6 +138,26 @@ The controller automatically removes all annotations with the
 
 - the Host is deprovisioned
 
+The API has been extended to permit either a hard or a soft reboot to
+take place, specified via the reboot annotation and a `mode` parameter. The API
+supports the following syntax:
+
+- `reboot.metal.io` -- immediate reboot via soft shutdown first,
+  followed by a hard shutdown if the soft shutdown fails.
+- `reboot.metal3.io: {'mode':'hard'}` -- immediate reboot via hard
+  shutdown, potentially allowing for high-availability use-cases.
+- `reboot.metal3.io/{key}` -- phased reboot, issued and managed by the
+  client registered with the `key`, via soft shutdown first, followed
+  by a hard reboot if the soft reboot fails.
+- `reboot.metal3.io:/{key} {'mode':'hard'}` -- phased reboot, issued
```

Comment:
I think the colons are in wrong place?
should be  `reboot.metal3.io/{key}: {'mode':'hard'}`

Reply:
Good catch, thank you!